apis/submariner/v1alpha1/servicediscovery_types.go

```diff
@@ -43,6 +43,7 @@ type ServiceDiscoverySpec struct {
 	// +listType=set
 	CustomDomains  []string          `json:"customDomains,omitempty"`
 	ImageOverrides map[string]string `json:"imageOverrides,omitempty"`
+	DeploymentInfo DeploymentInfo    `json:"deploymentInfo,omitempty"`
```

Comment:
This should be in the Status, and not the Spec, right?

I see there is not status on service discovery, we either can create it or, not add it.

Reply:
IIUC it exists - https://github.com/submariner-io/submariner-operator/blob/8388f77a46d7b1d45fe2d7fc204149e85053b827/apis/submariner/v1alpha1/servicediscovery_types.go#L58